tests/func/test_full.py

```diff
@@ -37,3 +37,77 @@ def test_run(psql):
     WHERE description = 'mail: alice@ldap2pg.docker';
     """)
     assert comment
+
+
+def test_role_config(psql):
+    # type: (PSQL) -> None
+
+    from sh import ldap2pg
+    c = 'tests/func/ldap2pg.config.yml'
+
+    roles = list(psql.roles())
+
+    # Test config_test_update before sync
+    assert 'config_test_update' in roles
+    update_config = psql.config('config_test_update')
+    expected_update_config = {
+        'log_statement': 'all',
+        'application_name': 'config_test_role_update_old_application_name',
+    }
+    assert update_config == expected_update_config
```

Comment:
I like the test, it covers everything we need.

The standard for assert is to put `expected` on left and `output` on right of the comparision. Could you swap the variables names to be consistent with this ? pytest diff output is more natural this way.